.github/workflows/ci-code.yml

```diff
@@ -96,15 +64,15 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     - name: Setup environment
-      run: .github/workflows/setup.sh
+      run: source .venv/bin/activate && .github/workflows/setup.sh
```

Comment:
Where is this `.venv` created from?

Reply:
It is created by `uv sync` automatically in the `install-aiida-core` action above.